hudi-sync/hudi-hive-sync/run_sync_tool.sh

```diff
@@ -49,6 +49,15 @@ fi
 HIVE_JACKSON=`ls ${HIVE_HOME}/lib/jackson-*.jar | tr '\n' ':'`
 HIVE_JARS=$HIVE_METASTORE:$HIVE_SERVICE:$HIVE_EXEC:$HIVE_JDBC:$HIVE_JACKSON
 
+HIVE_CALCITE=`ls ${HIVE_HOME}/lib/calcite-*.jar | tr '\n' ':'`
+if [ -n "$HIVE_CALCITE" ]; then
+    HIVE_JARS=$HIVE_JARS:$HIVE_CALCITE
+fi
+HIVE_LIBFB303=`ls ${HIVE_HOME}/lib/libfb303-*.jar | tr '\n' ':'`
+if [ -n "$HIVE_LIBFB303" ]; then
+    HIVE_JARS=$HIVE_JARS:$HIVE_LIBFB303
```

Comment:
Can you explain why we need these 2 kinds of jars ?

Reply:
I had the same problem, hive3 need these jars to be connected. The dependency packages of hive2 and hive3 on the connection have been changed.

Reply:
I had the same problem, hive3 need these jars to be connected. The dependency packages of hive2 and hive3 on the connection have been changed.

Reply:
Is it possible to distinguish between Hive2 and Hive3 and make different library dependency ?